<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>382</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>382</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>382</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>AVR Testing client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="outputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>191</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>381</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>AVR Controls</string>
     </attribute>
     <widget class="QPushButton" name="AskPosition">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Request current position.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Get position</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MoveToZero">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Order move to 0 position.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Move to zero</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
       </font>
      </property>
      <property name="text">
       <string> Steps:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MoveToPos">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Order to move forward or backward for specific steps count.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Move</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputSteps">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif</family>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Step count. Can be also lower than 0.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Connection data</string>
     </attribute>
     <widget class="QLineEdit" name="serverHost">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>localhost</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="serverPort">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>28338</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string> Host:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>62</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string> Port:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Co&amp;nnection</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <widget class="QMenu" name="menuLog">
    <property name="title">
     <string>&amp;Log</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="actionSave_to_file"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuConnection"/>
   <addaction name="menuLog"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="actionSave_to_file">
   <property name="text">
    <string>&amp;Save to file...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
